ui/pages/index.js

```diff
@@ -51,7 +52,9 @@ class Index extends PureComponent {
                   <LegacyI18nProvider>
                     <CurrencyRateProvider>
                       <MetamaskNotificationsProvider>
-                        <Routes />
+                        <TokenRatesProvider>
```

Comment:
We eventually want polling to be UI based, where UI components will call `useTokenRatesPolling` directly as their means of selecting the rates from state.   In the meantime this keeps the polling global, as it was before.

Reply:
My thought here is that we should create a single `PollingProvider` that can in turn manage all our polling contexts.

I'd ideally want us to avoid getting into a situation where we have a bunch of wrappers for each piece of state that needs to be polled.

I can imagine a world where we could still have the (single) overall polling provider be global, but then have each component hook into the polling context that it needs.

Reply:
I'm not sure exactly how that would look, and it's probably out of scope for PortfolioView deadline anyway. Wanted to put this thought out there though

Reply:
A single asset provider/context makes sense, while we're still polling globally.  I was thinking this as well.  I expect the providers/context to go away once everything is UI based and each component us calling the specific polling hooks it needs.

Reply:
Added AssetPollingProvider